content/docs/configuration/ca.md

```diff
@@ -97,6 +86,26 @@ ca-issuer     True    Signing CA verified   2m
 Certificates are now ready to be requested by using the CA `Issuer` named
 `ca-issuer` within the `sandbox` namespace.
 
+### Revocation Sources
+
+Optionally, you can specify [CRL](https://en.wikipedia.org/wiki/Certificate_revocation_list) Distribution Points or [OCSP servers](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol)
+for the CA issuer, to enable clients to check the revocation status of certificates issued by this CA.
+
+These are arrays of strings which give the location from which either a CRL or an OCSP response can be retrieved, respectively. The CA issuer will then include these in the issued certificates.
```

Comment:
"arrays of strings" reads a bit non-user-friendly to me. Isn't this list(s) of URLs?